app/views/content/is-teaching-right-for-me/behaviour-management.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Managing behaviour in the classroom"
+heading: "Managing behaviour in the classroom"
+subcategory: Qualifications and experience
```

Comment:
This feels a bit odd in terms of location - `Qualifications and experience` is what you need to have

I appreciate that we are creating this page before we create the proper section for which it should go in though!

Do we need a new blue subcategory for this page (maybe bring in the ECT page alongside it as that is orphaned currently)?

Reply:
I was struggling a bit as to where it should sit as I thought it should definitely be in the ITRFM section for the time being. So I think a new subcategory with the ECT page is a great idea.